app/routes/index.tsx

```diff
@@ -2,10 +2,14 @@ import { LinksFunction } from '@remix-run/node';
 import styles from 'react-datepicker/dist/react-datepicker.css';
 import Calendar from '~/components/Calendar';
 import Navbar from '~/components/common/navbar';
+import { useStore } from '~/store/useStore';
+import { EventState, UserState } from '~/utils/interfaces';
 
 export const links: LinksFunction = () => [{ rel: 'stylesheet', href: styles }];
 
 export default function Index() {
+  const events = useStore((state: EventState & UserState) => state.events);
+
```

Comment:
we don’t need this in `index.tsx`. We can directly import in the component where we need it.